docs/parallelism.md

```diff
@@ -0,0 +1,44 @@
+# Limiting parallelism
+
+You can restrict the number of workflows being executed at any time using these mechanisms.
```

Comment:
```suggestion
You can restrict the number of workflow executions.
```

- simplify

Reply:
I think the extra words here are important.
"Restrict the number of workflow executions." is not enough detail. Total number of executions ever?

I'll go with "You can restrict the number of simultaneous workflow executions."

Reply:
That was the exact word I was thinking to add 🙂👍